examples/datasource-http-backend/README.md

```diff
@@ -26,26 +32,52 @@ The plugin expects the following JSON from a remote HTTP API:
 }
 ```
 
+An example HTTP server that returns dummy data in this format is included in `cmd/server`.
+
+## Get started
+
+Data source plugins consist of both frontend and backend components. Install these components with the following CLI commands.
 ## Building
 
 ### Frontend
 
+Install dependencies and build:
+
 ```bash
 $ npm install
 $ npm run build
 ```
 
 ### Backend
 
+Build backend plugin binaries for Linux, Windows and Darwin:
+
 ```bash
 $ mage -v
 ```
 
-### Example server
+### Set up a server
 
+1. Set up a backend server to return data:
+
+
+```bash
+$ mage server
+$ ./cmd/server/server :10000
+2022/10/28 15:43:16 listening on :10000
+```
+
+2. Add a new data source in Grafana and use the following URL:
 The mockserver required for testing has been included in the Docker Compose file 
 
 Add a new data source in Grafana using the following URL:
 ```
 http://host.docker.internal:10000/metrics
 ```
+
```

Comment:
We are missing the instructions to run the docker container so one can see the plugin running `npm run server` do we want to add that?